apps/site/src/pages/index.astro

```diff
@@ -45,18 +41,31 @@ const projects = await getCollection('projects');
       </div>
     </div>
   </section>
-  <script is:inline define:vars={{ amount: projects.length }}>
-    /**
-     * @param {number} min
-     * @param {number} max
-     */
-    function randomInteger(min, max) {
-      return Math.floor(Math.random() * (max - min + 1)) + min;
-    }
+  <section>
+    <div class="max-w-7xl px-2 mx-auto py-16 lg:items-center">
+      <h1
+        class="font-extrabold uppercase text-3xl sm:text-4xl max-w-2xl font-heading"
+      >
+        Friends
+      </h1>
+      <div class="mt-12 grid sm:grid-cols-[repeat(2,1fr)] lg:grid-cols-[repeat(3,1fr)] items-center gap-4 items-stretch">
+        {friends.map((friend) => <FriendsCard {friend} />)}
+      </div>
+    </div>
```

Comment:
I have zero CSS skills, so I don't have a suggestion for how to do it. But I think it would be better to have the cards be on top of each other on smaller screens

![image](https://github.com/user-attachments/assets/587a4424-7bd5-41f0-8ce1-7ff2987fa274)

Reply:
Good catch! I can fix that real quick

Reply:
Actually looks like they are set to do that already on screen sizes < 640px
<img width="589" alt="image" src="https://github.com/user-attachments/assets/f742db55-e527-469e-8001-c3b7ddc533bf">

Reply:
@gingerchew maybe have another breakpoint?